openapi/api.github.com/operations/repos/create-dispatch-event.json

```diff
@@ -66,8 +66,18 @@
             "event_type": {
               "type": "string",
               "description": "**Required:** A custom webhook event name."
+            },
+            "client_payload": {
+              "type": "object",
+              "description": "JSON payload with extra information about the webhook event that your action or worklow may use.",
+              "default": "",
+              "properties": {}
```

Comment:
any properties are allowed, I'll have to look up how that looks like in JSON Schema

Reply:
okay turns out that allowing any properties with any values is the default behavior